buildSrc/plugins/src/main/kotlin/org/jetbrains/kotlin/FrameworkTest.kt

```diff
@@ -85,9 +85,21 @@ open class FrameworkTest : DefaultTask() {
     }
 
     private fun runTest(testExecutable: Path) {
+        val target = project.testTarget()
+        val platform = project.platformManager().platform(target)
+        assert(platform.configurables is AppleConfigurables)
```

Comment:
What's the motivation behind asserting the condition to be checked by `as` at next line?

Reply:
Not much, I guess.